From 41700ce3d08549d27f7cbcb17fa88f12d4238e98 Mon Sep 17 00:00:00 2001
From: Yorick Peterse <yorick@yorickpeterse.com>
Date: Thu, 19 Dec 2024 21:13:35 +0100
Subject: [PATCH] Patch perfparser to handle newer perf versions

---
 3rdparty/perfparser/app/perffeatures.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/3rdparty/perfparser/app/perffeatures.cpp b/3rdparty/perfparser/app/perffeatures.cpp
index a52e102..0fe4e31 100644
--- a/3rdparty/perfparser/app/perffeatures.cpp
+++ b/3rdparty/perfparser/app/perffeatures.cpp
@@ -150,6 +150,8 @@ QDataStream &operator>>(QDataStream &stream, PerfBuildId &buildId)
     while (next < buildId.size) {
         PerfEventHeader header;
         stream >> header;
+        if(!header.size)
+          break;

         PerfBuildId::BuildId build;
         stream >> build.pid;
--
2.47.1

