--- stylua-2.4.1/Cargo.toml	2006-07-24T01:21:28+00:00
+++ stylua-2.4.1/Cargo.toml	2026-04-14T01:48:10.379349+00:00
@@ -47,7 +47,6 @@
 ]
 default = [
     "editorconfig",
-    "wasm-bindgen",
     "lsp",
 ]
 editorconfig = ["ec4rs"]
@@ -198,9 +197,6 @@
 version = "1.34.0"
 features = ["glob"]
 
-[target.'cfg(target_arch = "wasm32")'.dependencies.wasm-bindgen]
-version = "0.2.108"
-optional = true
-
 [profile.dev.package.full_moon]
 opt-level = 3
+
